### ISSUE TITLE
[Merge]

### ISSUE THREAD

**Issue by jdesaphy** [monarch-initiative/mondo#5458]
Would it be possible to merge those pairs together please? 

NCIT_C12664 | Abdomen | UBERON_0000916 | abdomen
MONDO_0005227 | abscess | MPATH_608 | abscess
HP_0001735 | Acute pancreatitis | MONDO_0006515 | acute pancreatitis
HP_0010524 | Agnosia | MONDO_0005638 | agnosia
HP_0012234 | Agranulocytosis | MONDO_0001609 | agranulocytosis
HP_0001022 | Albinism | MONDO_0043209 | albinism
HP_0001596 | Alopecia | MONDO_0004907 | alopecia
HP_0000646 | Amblyopia | MONDO_0001020 | amblyopia
CHEBI_16134 | ammonia | NCBITaxon_29189 | Ammonia
HP_0007354 | Amyotrophic lateral sclerosis | MONDO_0004976 | amyotrophic lateral sclerosis
CARO_0000000 | anatomical entity | UBERON_0001062 | anatomical entity
MONDO_0002280 | anemia | NCBITaxon_12939 | Anemia
HP_0001903 | Anemia | MONDO_0002280 | anemia
HP_0001903 | Anemia | NCBITaxon_12939 | Anemia
HP_0000970 | Anhidrosis | MONDO_0006527 | anhidrosis
HP_0000526 | Aniridia | MONDO_0019172 | aniridia
HP_0000458 | Anosmia | MONDO_0010528 | anosmia
HP_0000739 | Anxiety | MONDO_0011918 | anxiety
HP_0001650 | Aortic valve stenosis | MONDO_0042981 | aortic valve stenosis
HP_0002186 | Apraxia | MONDO_0000665 | apraxia
HP_0010527 | Astereognosia | MONDO_0000680 | astereognosia
HP_0002099 | Asthma | MONDO_0004979 | asthma
HP_0000717 | Autism | MONDO_0005260 | autism
HP_0010885 | Avascular necrosis | MONDO_0018373 | avascular necrosis
HP_0000027 | Azoospermia | MONDO_0100459 | azoospermia
GO_0007610 | behavior | NCIT_C16326 | Behavior
HP_0005912 | Biliary atresia | MONDO_0008867 | biliary atresia
HP_0001156 | Brachydactyly | MONDO_0021004 | brachydactyly
HP_0002110 | Bronchiectasis | MONDO_0004822 | bronchiectasis
HP_0001638 | Cardiomyopathy | MONDO_0004994 | cardiomyopathy
GO_0003824 | catalytic activity | PATO_0001414 | catalytic activity
HP_0000518 | Cataract | MONDO_0005129 | cataract
HP_0100658 | Cellulitis | MONDO_0005230 | cellulitis
HP_0030724 | Central nervous system cyst | MONDO_0005262 | central nervous system cyst
HP_0002438 | Cerebellar malformation | MONDO_0015915 | cerebellar malformation
HP_0007313 | Cerebral degeneration | MONDO_0024238 | cerebral degeneration
HP_0030159 | Cervical polyp | MONDO_0000751 | cervical polyp
HP_0001396 | Cholestasis | MONDO_0001751 | cholestasis
HP_0009797 | Cholesteatoma | MONDO_0006530 | cholesteatoma
HP_0010762 | Chordoma | MONDO_0008978 | chordoma
HP_0005558 | Chronic leukemia | MONDO_0001014 | chronic leukemia
HP_0002257 | Chronic rhinitis | MONDO_0004514 | chronic rhinitis
PATO_0000411 | circular | SO_0000988 | circular
HP_0000175 | Cleft palate | MONDO_0016064 | cleft palate
HP_0005387 | Combined immunodeficiency | MONDO_0015131 | combined immunodeficiency
HP_0004431 | Complement deficiency | MONDO_0003832 | complement deficiency
HP_0001635 | Congestive heart failure | MONDO_0005009 | congestive heart failure
HP_0000509 | Conjunctivitis | MONDO_0003799 | conjunctivitis
HP_0002812 | Coxa vara | MONDO_0007391 | coxa vara
HP_0031258 | Delirium | MONDO_0045057 | delirium
HP_0000726 | Dementia | MONDO_0001627 | dementia
HP_0001651 | Dextrocardia | MONDO_0015661 | dextrocardia
HP_0000819 | Diabetes mellitus | MONDO_0005015 | diabetes mellitus
NCIT_C12378 | Digestive System | UBERON_0001007 | digestive system
HP_0001644 | Dilated cardiomyopathy | MONDO_0005021 | dilated cardiomyopathy
HP_0010522 | Dyslexia | MONDO_0005489 | dyslexia
ENVO_00000176 | elevation | PATO_0001687 | elevation
HP_0002383 | Encephalitis | MONDO_0019956 | encephalitis
HP_0100584 | Endocarditis | MONDO_0005025 | endocarditis
MFOMD_0000105 | enuresis | MONDO_0024290 | enuresis
HP_0000031 | Epididymitis | MONDO_0004779 | epididymitis
HP_0100639 | Erectile dysfunction | MONDO_0005362 | erectile dysfunction
HP_0010525 | Finger agnosia | MONDO_0000671 | finger agnosia
CHEBI_64047 | food additive | FOODON_03412972 | food additive
HP_0002145 | Frontotemporal dementia | MONDO_0017276 | frontotemporal dementia
HP_0009064 | Generalized lipodystrophy | MONDO_0027766 | generalized lipodystrophy
HP_0000230 | Gingivitis | MONDO_0002508 | gingivitis
HP_0000501 | Glaucoma | MONDO_0005041 | glaucoma
HP_0000099 | Glomerulonephritis | MONDO_0002462 | glomerulonephritis
HP_0001952 | Glucose intolerance | MONDO_0001076 | glucose intolerance
HP_0000853 | Goiter | MONDO_0005397 | goiter
HP_0000133 | Gonadal dysgenesis | MONDO_0001967 | gonadal dysgenesis
HP_0001997 | Gout | MONDO_0005393 | gout
HP_0010566 | Hamartoma | MONDO_0006499 | hamartoma
HP_0001878 | Hemolytic anemia | MONDO_0003664 | hemolytic anemia
HP_0012115 | Hepatitis | MONDO_0002251 | hepatitis
HP_0002156 | Homocystinuria | MONDO_0004737 | homocystinuria
HP_0010606 | Hordeolum | MONDO_0005800 | hordeolum
HP_0000238 | Hydrocephalus | MONDO_0001150 | hydrocephalus
HP_0001789 | Hydrops fetalis | MONDO_0015193 | hydrops fetalis
HP_0000859 | Hyperaldosteronism | MONDO_0003009 | hyperaldosteronism
HP_0002904 | Hyperbilirubinemia | MONDO_0024288 | hyperbilirubinemia
HP_0003074 | Hyperglycemia | MONDO_0002909 | hyperglycemia
HP_0003077 | Hyperlipidemia | MONDO_0021187 | hyperlipidemia
HP_0010980 | Hyperlipoproteinemia | MONDO_0037748 | hyperlipoproteinemia
HP_0100774 | Hyperostosis | MONDO_0002185 | hyperostosis
HP_0000843 | Hyperparathyroidism | MONDO_0001741 | hyperparathyroidism
HP_0002905 | Hyperphosphatemia | MONDO_0000328 | hyperphosphatemia
HP_0000953 | Hyperpigmentation of the skin | MONDO_0019289 | hyperpigmentation of the skin
HP_0010514 | Hyperpituitarism | MONDO_0006793 | hyperpituitarism
HP_0100786 | Hypersomnia | MONDO_0005466 | hypersomnia
HP_0000998 | Hypertrichosis | MONDO_0019280 | hypertrichosis
HP_0001931 | Hypochromic anemia | MONDO_0001357 | hypochromic anemia
HP_0001943 | Hypoglycemia | MONDO_0004946 | hypoglycemia
HP_0000135 | Hypogonadism | MONDO_0002146 | hypogonadism
HP_0000044 | Hypogonadotropic hypogonadism | MONDO_0018555 | hypogonadotropic hypogonadism
HP_0000966 | Hypohidrosis | MONDO_0006560 | hypohidrosis
HP_0000829 | Hypoparathyroidism | MONDO_0001220 | hypoparathyroidism
HP_0002148 | Hypophosphatemia | MONDO_0000313 | hypophosphatemia
HP_0001010 | Hypopigmentation of the skin | MONDO_0019290 | hypopigmentation of the skin
HP_0040075 | Hypopituitarism | MONDO_0005152 | hypopituitarism
CARO_0000007 | immaterial anatomical entity | UBERON_0000466 | immaterial anatomical entity
HP_0001249 | Intellectual disability | MONDO_0001071 | intellectual disability
NCIT_C12736 | Intestine | UBERON_0000160 | intestine
HP_0000491 | Keratitis | MONDO_0003085 | keratitis
HP_0001096 | Keratoconjunctivitis | MONDO_0004768 | keratoconjunctivitis
HP_0003128 | Lactic acidosis | MONDO_0006040 | lactic acidosis
HP_0004789 | Lactose intolerance | MONDO_0100345 | lactose intolerance
HP_0030682 | Left ventricular noncompaction | MONDO_0018901 | left ventricular noncompaction
HP_0001909 | Leukemia | MONDO_0005059 | leukemia
HP_0001882 | Leukopenia | MONDO_0003785 | leukopenia
HP_0009125 | Lipodystrophy | MONDO_0006573 | lipodystrophy
HP_0002840 | Lymphadenitis | MONDO_0002052 | lymphadenitis
HP_0001004 | Lymphedema | MONDO_0019297 | lymphedema
HP_0001888 | Lymphopenia | MONDO_0003783 | lymphopenia
Orphanet_93448 | Lysosomal storage disease with skeletal involvement | MONDO_0800088 | lysosomal storage disease with skeletal involvement
HP_0001972 | Macrocytic anemia | MONDO_0002281 | macrocytic anemia
HP_0000608 | Macular degeneration | MONDO_0003004 | macular degeneration
CARO_0000006 | material anatomical entity | UBERON_0000465 | material anatomical entity
HP_0001889 | Megaloblastic anemia | MONDO_0001700 | megaloblastic anemia
HP_0001287 | Meningitis | MONDO_0021108 | meningitis
HP_0002435 | Meningocele | MONDO_0001147 | meningocele
MFOMD_0000004 | mental disorder | MONDO_0005084 | mental disorder
HP_0001942 | Metabolic acidosis | MONDO_0000440 | metabolic acidosis
HP_0002912 | Methylmalonic acidemia | MONDO_0002012 | methylmalonic acidemia
HP_0000252 | Microcephaly | MONDO_0001149 | microcephaly
HP_0001935 | Microcytic anemia | MONDO_0001245 | microcytic anemia
HP_0000568 | Microphthalmia | MONDO_0021129 | microphthalmia
CARO_0010000 | multicellular anatomical structure | UBERON_0010000 | multicellular anatomical structure
HP_0003560 | Muscular dystrophy | MONDO_0020121 | muscular dystrophy
HP_0002300 | Mutism | MONDO_0002905 | mutism
HP_0011974 | Myelofibrosis | MONDO_0044903 | myelofibrosis
HP_0002475 | Myelomeningocele | MONDO_0019773 | myelomeningocele
HP_0003198 | Myopathy | MONDO_0005336 | myopathy
HP_0030050 | Narcolepsy | MONDO_0021107 | narcolepsy
HP_0002664 | Neoplasm | MONDO_0005070 | neoplasm
HP_0000123 | Nephritis | MONDO_0001166 | nephritis
HP_0000090 | Nephronophthisis | MONDO_0019005 | nephronophthisis
HP_0045005 | Neural tube defect | MONDO_0018075 | neural tube defect
CARO_0001001 | neuron projection bundle | UBERON_0000122 | neuron projection bundle
HP_0000648 | Optic atrophy | MONDO_0003608 | optic atrophy
NCIT_C13018 | Organ | UBERON_0000062 | organ
HP_0002754 | Osteomyelitis | MONDO_0005246 | osteomyelitis
HP_0011002 | Osteopetrosis | MONDO_0017198 | osteopetrosis
HP_0000939 | Osteoporosis | MONDO_0005298 | osteoporosis
HP_0001733 | Pancreatitis | MONDO_0004982 | pancreatitis
HP_0001876 | Pancytopenia | MONDO_0001529 | pancytopenia
HP_0010550 | Paraplegia | MONDO_0003757 | paraplegia
HP_0001698 | Pericardial effusion | MONDO_0001370 | pericardial effusion
HP_0009830 | Peripheral neuropathy | MONDO_0005244 | peripheral neuropathy
EFO_0020005 | pheochromocytoma-paraganglioma | MONDO_0035540 | pheochromocytoma-paraganglioma
HP_0002090 | Pneumonia | MONDO_0005249 | pneumonia
HP_0010442 | Polydactyly | MONDO_0021003 | polydactyly
HP_0000826 | Precocious puberty | MONDO_0000088 | precocious puberty
HP_0008207 | Primary adrenal insufficiency | MONDO_0015128 | primary adrenal insufficiency
Orphanet_93444 | Primary bone dysplasia with increased bone density | MONDO_0800084 | primary bone dysplasia with increased bone density
PO_0025127 | primordium | UBERON_0001048 | primordium
HP_0010528 | Prosopagnosia | MONDO_0003227 | prosopagnosia
HP_0000093 | Proteinuria | MONDO_0003634 | proteinuria
HP_0000508 | Ptosis | MONDO_0000728 | ptosis
HP_0100598 | Pulmonary edema | MONDO_0006932 | pulmonary edema
HP_0002204 | Pulmonary embolism | MONDO_0005279 | pulmonary embolism
HP_0000979 | Purpura | MONDO_0002610 | purpura
HP_0000999 | Pyoderma | MONDO_0002922 | pyoderma
BFO_0000019 | quality | PATO_0000001 | quality
HP_0002035 | Rectal prolapse | MONDO_0004754 | rectal prolapse
GO_0050795 | regulation of behavior | NBO_0001786 | regulation of behavior
GO_0019098 | reproductive behavior | NBO_0000010 | reproductive behavior
HP_0000546 | Retinal degeneration | MONDO_0004580 | retinal degeneration
HP_0001370 | Rheumatoid arthritis | MONDO_0008383 | rheumatoid arthritis
HP_0012384 | Rhinitis | MONDO_0003014 | rhinitis
HP_0002748 | Rickets | MONDO_0005520 | rickets
BFO_0000023 | role | CHEBI_50906 | role
HP_0100324 | Scleroderma | MONDO_0019340 | scleroderma
HP_0001051 | Seborrheic dermatitis | MONDO_0006608 | seborrheic dermatitis
HP_0001153 | Septate vagina | MONDO_0015848 | septate vagina
HP_0004430 | Severe combined immunodeficiency | MONDO_0015974 | severe combined immunodeficiency
HP_0001924 | Sideroblastic anemia | MONDO_0015194 | sideroblastic anemia
HP_0002652 | Skeletal dysplasia | MONDO_0018230 | skeletal dysplasia
GO_0035176 | social behavior | NBO_0000011 | social behavior
HP_0001328 | Specific learning disability | MONDO_0016225 | specific learning disability
HP_0002414 | Spina bifida | MONDO_0008449 | spina bifida
11359 | STAR | ENVO_01000801 | star
HP_0010280 | Stomatitis | MONDO_0004842 | stomatitis
HP_0000486 | Strabismus | MONDO_0003432 | strabismus
HP_0004381 | Supravalvular aortic stenosis | MONDO_0008504 | supravalvular aortic stenosis
HP_0001159 | Syndactyly | MONDO_0021002 | syndactyly
HP_0002725 | Systemic lupus erythematosus | MONDO_0007915 | systemic lupus erythematosus
HP_0001873 | Thrombocytopenia | MONDO_0002049 | thrombocytopenia
HP_0009804 | Tooth agenesis | MONDO_0005486 | tooth agenesis
HP_0100603 | Toxemia of pregnancy | MONDO_0045048 | toxemia of pregnancy
HP_0001669 | Transposition of the great arteries | MONDO_0000153 | transposition of the great arteries
HP_0000243 | Trigonocephaly | MONDO_0000156 | trigonocephaly
HP_0001660 | Truncus arteriosus | UBERON_0002061 | truncus arteriosus
HP_0006000 | Ureteral obstruction | MONDO_0003329 | ureteral obstruction
HP_0001025 | Urticaria | MONDO_0005492 | urticaria
HP_0000554 | Uveitis | MONDO_0020283 | uveitis
HP_0002633 | Vasculitis | MONDO_0018882 | vasculitis
HP_0005293 | Venous insufficiency | MONDO_0000945 | venous insufficiency
HP_0001663 | Ventricular fibrillation | MONDO_0000190 | ventricular fibrillation
HP_0006562 | Viral hepatitis | MONDO_0006011 | viral hepatitis
HP_0030222 | Visual agnosia | MONDO_0000685 | visual agnosia
HP_0100502 | Vitamin B12 deficiency | MONDO_0020696 | vitamin B12 deficiency
HP_0000991 | Xanthomatosis | MONDO_0002615 | xanthomatosis

**Comment by nicolevasilevsky** [comment]
Hi @jdesaphy 

Orphanet:93444 Primary bone dysplasia with increased bone density is already an equivalent database cross reference on MONDO:0800084  primary bone dysplasia with increased bone density.

and Orphanet:93448 Lysosomal storage disease with skeletal involvement is an equivalent database cross reference on MONDO_0800088 lysosomal storage disease with skeletal involvement.

I will add EFO:0020005  pheochromocytoma-paraganglioma as an xref to MONDO:0035540  pheochromocytoma-paraganglioma.

We cannot merge the other terms together, as they are from different ontologies. For example, a term in HPO is intended to represent the phenotypic form of the term and Mondo represents the disease form. 

On our [FAQ page](https://mondo.monarchinitiative.org/pages/faq/), we describe the difference between a disease and a phenotype:

There is a lot of overlap between diseases and phenotypes. Mondo and Human Phenotype Ontology (HPO) define a disease as an entity that has all four of the following features:

an etiology (whether identified or as yet unknown)
a time course
a set of phenotypic features
if treatments exist, there is a characteristic response to them
A phenotype (or a phenotypic feature) is a component of a disease, and HPO terms can be used to describe the set of phenotypic features that characterize a disease. There is a grey zone between diseases and phenotypic features. For instance, diabetes mellitus can be conceptualized as a disease, but it is also a feature of other diseases such as Bardet Biedl syndrome. In some cases like this, Mondo may label the disease entity with (disease) after the disease name, to distinguish it from the HPO phenotype term (for example, [MONDO:0005466 ‘hypersomnia (disease)’](http://www.ontobee.org/ontology/MONDO?iri=http://purl.obolibrary.org/obo/MONDO_0005466) and [HP:0100786 Hypersomnia](https://hpo.jax.org/app/browse/term/HP:0100786)).

--
While these terms listed above have the same label, they do not necessarily have the same meaning. For example, MONDO:0002280  anemia is a disease and NCBITaxon:12939 Anemia is a plant.

Please let me know if you'd like to discuss further, otherwise I will close this ticket. Thanks!
